docs/index.md

```diff
@@ -1,5 +1,6 @@
 # OCFL Community Extensions
 
+  * [_initial_ Placeholder](initial.md)
```

Comment:
Per @zimeon's [comment](https://github.com/OCFL/extensions/pull/74#pullrequestreview-1911796064), the suggestion is to name this: "initial: Initial Extension" or "initial".